routes/contract.js

```diff
@@ -32,12 +56,51 @@ router.post("/create", async (req, res) => {
   }
 });
 
+/** /contract/details/:contractNo */
+router.get("/details/:contractNo", async (req, res) => {
+  try {
+    let contractNo = req.params.contractNo;
+    let contractContent = await getContractContentByContractNo(contractNo);
+    return res.status(201).json({ status: 'success', message: 'Get contract content successfully', contractContent });
+  } catch (err) {
+    console.error(`Error while getting contract content`);
+    console.error(err);
+    return res.status(500).json({ status : 'error', message : err.message });
+  }
+});
+
+/** /contract/details/:contractNo */
+router.post("/details/:contractNo", async (req, res) => {
+  try {
+    let contractNo = req.params.contractNo;
+    let secondParty = req.body.secondPartySignature;
+    await updateSecondPartySignature(contractNo, secondParty);
+    let updatedContent = await updateSecondParty(contractNo, secondParty);
+    return res.status(201).json({
```

Comment:
Please change this to 200
Can you please update everywhere 201 to 200 if you are not creating anything